.github/workflows/ci-home.yml

```diff
@@ -0,0 +1,69 @@
+name: CI - Home
+on:
+  push:
+    branches: [main]
+    paths:
+      - '.github/workflows/ci-home.yml'
+      - '**/home-manager/**.nix'
+      - 'Makefile.toml'
+  pull_request:
+    paths:
+      - '.github/workflows/ci-home.yml'
+      - '**/home-manager/**.nix'
+      - 'Makefile.toml'
+  schedule:
+    # Every 10:42 JST
+    # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule
+    - cron: '42 1 * * *'
+  workflow_dispatch:
+
+jobs:
+  home-manager:
+    timeout-minutes: 30
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-22.04
+          - macos-13
+        home-manager-channel:
+          - release-23.05
+          - master # unstable nixpkgs
```

Comment:
Preparation for updating to unstable nixpkgs https://github.com/kachick/dotfiles/issues/147#issuecomment-1655038956